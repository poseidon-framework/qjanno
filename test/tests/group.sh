qhs -H -O "SELECT foo,COUNT(*) cnt FROM big.csv GROUP BY foo ORDER BY cnt DESC LIMIT 3"
