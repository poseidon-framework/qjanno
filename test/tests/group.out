foo cnt
a0 4
c1 3
b1 2
